test/test_transforms.py

```diff
@@ -673,3 +677,128 @@ def test_bert_tokenizer_save_load(self, test_scripting, do_lower_case, return_to
             torch.save(tokenizer, tokenizer_path)
             loaded_tokenizer = torch.load(tokenizer_path)
             self._bert_tokenizer((loaded_tokenizer), do_lower_case=do_lower_case)
+
+
+class TestRegexTokenizer(TorchtextTestCase):
+    # TODO(Nayef211): remove decorator once	https://github.com/pytorch/pytorch/issues/38207 is closed
+    @unittest.skipIf(platform.system() == "Windows", "Test is known to fail on Windows.")
+    def test_RegexTokenizer(self):
```

Comment:
Let's actually change this to be `test_regex_tokenizer` to be consistent with how we name our other tests in this file.